src/utils/content_protection_utils/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+cmake_minimum_required (VERSION 3.5)
+
+project (content_protection_utils)
+
+add_library (${PROJECT_NAME} STATIC src/content_protection_utils.c)
+add_library (aduc::${PROJECT_NAME} ALIAS ${PROJECT_NAME})
+
+target_include_directories (${PROJECT_NAME} PUBLIC inc)
+
+find_package (azure_c_shared_utility REQUIRED)
+
+#
+# Turn -fPIC on, in order to use this library in another shared library.
+#
+set_property (TARGET ${PROJECT_NAME} PROPERTY POSITION_INDEPENDENT_CODE ON)
+
+target_link_libraries (
+    ${PROJECT_NAME}
+    PUBLIC aduc::c_utils aziotsharedutil aduc::crypto_lib
+    PRIVATE )
+
+# Always support test root keys.
+add_definitions (-DBUILD_WITH_TEST_KEYS=1)
```

Comment:
always? seems concerning to  me.

Reply:
Bug tracking fix: https://microsoft.visualstudio.com/OS/_queries/edit/43774392/?triage=true

It is. This is leftover from our service/client integration in the older days. We're working on fixing this now